php-practice/section2.php

```diff
@@ -117,6 +139,15 @@ function printNewline() {
 // HINT: Use a foreach loop
 // =======================================================================
 
+$sumArray = array("red", "green", "blue", "yellow"); 
```

Comment:
Let's change this array so all of the elements are integers instead of strings. That way it makes more sense to sum it, since summing just means adding together.

`array_sum` is essentially the answer to this problem. You could just do:

```
$sumArray = array(1, 2, 3, 4);
echo array_sum($sumArray);
```

So I'm essentially asking you to create your own version of `array_sum`. So try to answer this without using `array_sum`.